### ISSUE TITLE
Request support for Nedis portable air conditioner

### ISSUE THREAD

**Issue by Xpl0jd** [make-all/tuya-local#784]
# Log Message

```
Logger: custom_components.tuya_local.config_flow
Source: custom_components/tuya_local/config_flow.py:49
Integration: Tuya Local (documentation, issues)
First occurred: June 10, 2023 at 15:16:00 (4 occurrences)
Last logged: 10:39:09

Device matches mirabella_genio_usb with quality of 15%. DPS: {"updated_at": 1686402959.647889, "1": true, "2": 24, "3": 25, "4": "cold", "5": "med", "11": false, "15": "off", "20": 0, "103": false, "104": 0, "105": 0, "109": 18, "110": false}
Report this to https://github.com/make-all/tuya-local/issues/
Device matches mirabella_genio_usb with quality of 15%. DPS: {"updated_at": 1686472748.8957763, "1": false, "2": 22, "3": 23, "4": "cold", "5": "med", "11": false, "15": "off", "20": 0, "103": false, "104": 0, "105": 0, "109": 18, "110": false}

```

# Information about DPS mappings

```
dps
1:false
103:false
104:0
105:0
107:61
108:32
109:18
11:false
110:false
15:off
2:20
20:0
3:33
4:wind
5:med

Switch
code: "1"

Set temperature
code: "2"

Temperature
code: "3"

Mode:
code: "4"

Fan speed
code: "5"

Sleep
code: "103"

Switch timer on
code: "104"

Switch timer off
code: "105"

Ionisation
code: "11"

Set temp(F)
code: "107"

Temperature(F)
code: "108"

功能标记
code: "109"

Horizontal swing
code: "110"

Horizontal swing
code: "15"

Error alarm
code: "20"


```

# Product ID
"product_id": "vrredpnf22yayvhi"

# Information about how the device functions

Air Conditioner Functions
- Turn ON/OFF
- Measure real time Temp
- Set target temp for AC
- Change between 3 Modes, "Cool, Dry, Fan"
- Set fan speed "low, medium, high"
- Activate Automatic swing function for change airflow direction

I understand i give very bad and crappy information since im new to the whole Smart Home scene and still trying to learn.
I hope i did figure out what was needed, otherwise please explain what im looking for.

Here is the device sold in Sweden
https://www.webhallen.com/se/product/306482-Nedis-Barbar-luftkonditionering-AC-12-000-BTU-WIFI-Energiklass-A-Fjarrkontroll-Timerfunk

This is the Company homepage of the device.
https://nedis.co.uk/en-gb/product/home-living/climate/air-conditioners/550681737/mobile-air-conditioner-12000-btu-100-m3-3-speed-remote-control-shut-off-timer-white

**Comment by Xpl0jd** [comment]
Maybe this is usefull aswell.
switch			Boolean		"{true,false}"
temp_set		        Integer		{  "unit": "C",  "min": 16,  "max": 31,  "scale": 0,  "step": 1}
temp_current	        Integer		{  "unit": "C",  "min": 0,  "max": 35,  "scale": 0,  "step": 1}
mode			Enum		{  "range": [    "cold",    "wet",    "wind",    "hot"  ]}
anion			Boolean		"{true,false}"
temp_set_f		Integer		{  "unit": "F",  "min": 61,  "max": 88,  "scale": 0,  "step": 1}
temp_current_f	Integer		{  "unit": "F",  "min": 32,  "max": 95,  "scale": 0,  "step": 1}

**Comment by Zelaf** [comment]
This device had missing configuration for `swing_mode`. I managed to figure out that it was ID 110 and added the correct value mappings. 

I'm unsure what the other `swing_1` is for, or some of the other IDs but with this configuration change all functionality from the Tuya app has been added to Home Assistant!

```yaml
    - id: 110
      type: boolean
      name: swing_mode
      mapping:
        - dps_val: true
          value: "vertical"
        - dps_val: false
          value: "off"
```

Now I finally don't have to open the Tuya app to enable the swing function every time I need to start the AC!

A thing to note however, both in the Tuya app and the Tuya Platform API info, it's mentioned that the swinging is `horizontal` but it is in fact vertical on the unit itself, that to say the blades swing up and down and not side to side. So I decided to mark it as such in the configuration.
